packages/devextreme-schematics/src/add-view/index.ts

```diff
@@ -39,6 +39,14 @@ function getPathToFile(host: Tree, projectName: string, moduleName: string) {
   }
 }
 
+function capitalizeFirstLetter(name: string, startDelimiter: string, endDelimiter: string) {
```

Comment:
It looks like a duplication of this [humaize](https://github.com/DevExpress/devextreme-cli/blob/master/packages/devextreme-cli/src/utility/string.js#L5) method. @dxvladislavvolkov What do you think? Could we avoid this duplication?

Reply:
Yes, I deleted my method. Thank you.